usaspending_api/etl/subaward_etl.py

```diff
@@ -180,9 +180,16 @@ def load_subawards(submission_attributes, db_cursor):
         recipient = LegalEntity.objects.filter(recipient_unique_id=row['duns']).first()
 
         if not recipient and row['duns']:
+            recipient_name = row['awardee_name']
+            if recipient_name is None:
```

Comment:
yeah, because perhaps the recipient record we're linking to has the name. Or we could add something later that goes through and adds names to all recipients with a DUNS but with an empty name.